<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quiz App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="timerLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Time: 00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Question will appear here</string>
    </property>
   </widget>
   <widget class="QWidget" name="radioButtonsContainer" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>400</width>
      <height>120</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="option1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>30</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Option 1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="option2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>400</width>
       <height>30</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Option 2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="option3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>400</width>
       <height>30</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Option 3</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="option4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>400</width>
       <height>30</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Option 4</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="answerInput">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>30</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Type your answer here</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Next Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart Quiz</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>440</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Score: 0 / 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
